app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments, class_name: 'Comment', foreign_key: :post_id
+  has_many :likes, class_name: 'Like', foreign_key: :post_id
+
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_post_count
+    author.increment!(:posts_counter)
+  end
+
+  def recent_comments
+    Comment.order(created_at: :desc).limit(5)
```

Comment:
- [ ] We wish to return the five most recent` comments` A post has many `comments` 👍 . Check the comment above for more information: **Hints**

`comments.order('created_at Desc').limit(5)`